src/Propagation/TraceResponse/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "open-telemetry/opentelemetry-propagation-traceresponse",
```

Comment:
I'm happy with this name. I've created https://github.com/opentelemetry-php/contrib-propagator-traceresponse which will be the git-split target for this directory. Can you add an entry to the top-level `.gitsplit.yaml` file for this dir. After this is merged, I'll add it to packagist and tag.